designer/server/src/main/scala/pl/touk/nussknacker/ui/server/AkkaHttpBasedRouteProvider.scala

```diff
@@ -490,11 +491,12 @@ class AkkaHttpBasedRouteProvider(
         designerClock
       )
 
-      val statisticUrlConfig = if (usageStatisticsReportsConfig.encryptionEnabled) {
-        StatisticUrlConfig(maybePublicEncryptionKey = Some(PublicEncryptionKey.INSTANCE))
-      } else {
-        StatisticUrlConfig()
-      }
+      val publicKey =
+        Using(Source.fromURL(getClass.getResource("/encryption.key"))) { read =>
+          read.mkString.trim
+        }.get
+
+      val statisticUrlConfig = StatisticUrlConfig(maybePublicEncryptionKey = Some(PublicEncryptionKey(publicKey)))
```

Comment:
What do you think about removing an option from `StatisticUrlConfig.maybePublicEncryptionKey`?